ui/app/serializers/clients/activity.js

```diff
@@ -18,55 +21,11 @@ export default class ActivitySerializer extends ApplicationSerializer {
       response_timestamp,
       by_namespace: formatByNamespace(payload.data.by_namespace),
       by_month: formatByMonths(payload.data.months),
-      total: homogenizeClientNaming(payload.data.total),
+      total: destructureClientCounts(payload.data.total),
     };
     delete payload.data.by_namespace;
     delete payload.data.months;
     delete payload.data.total;
     return super.normalizeResponse(store, primaryModelClass, transformedPayload, id, requestType);
   }
 }
-/*
-SAMPLE PAYLOAD BEFORE/AFTER:
```

Comment:
I removed the scattered samples (which were also outdated) in favor of a single sample use in `tests/helpers/clients`